x-pack/plugins/security_solution/server/lib/telemetry/filterlists/endpoint_alerts.ts

```diff
@@ -103,6 +112,7 @@ const allowlistBaseEventFields: AllowlistFields = {
     id: true,
   },
   Persistence: true,
+  Effective_process: true,
```

Comment:
lint error  Object Literal Property name `Effective_process` must match one of the following formats: camelCase, PascalCase, snake_case, UPPER_CASE  @typescript-eslint/naming-convention

Field name is `Effective_process`, and can't be changed to satisfy one of the above formats.

https://github.com/elastic/endpoint-dev/blob/main/Python/endpoint/generators/events/v0/events.yml#L535